internal/cli/branding.go

```diff
@@ -138,17 +139,17 @@ func showBrandingCmd(cli *cli) *cobra.Command {
 		Long:    "Display the custom branding settings for Universal Login.",
 		Example: "auth0 branding show",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			var branding *management.Branding // Load app by id
+			var myBranding *management.Branding // Load app by id
```

Comment:
Any particular reason to rename this to `myBranding`?

Reply:
Unfortunately it was conflicting with the `github.com/auth0/auth0-cli/internal/branding` package name.